tests/tests.cabal

```diff
@@ -94,11 +100,10 @@ executable benchmark-stitch-lh
                       -Wno-unticked-promoted-constructors
                       -fno-warn-name-shadowing
                       -fkeep-going
-                      -fforce-recomp
-                      -ddump-timings
-                      -ddump-to-file
                       -O0
 
+    if flag(measure-timings)
+      ghc-options:    -fforce-recomp -ddump-timings -ddump-to-file
```

Comment:
In my experience with playing with cabal, `-fforce-recomp` only works the first time you add it as a ghc flag (because the ghc-flags have changed) and doesn't have an effect after that.  Have you tried running the test-suite twice after this change to make sure the dumps are still picked up, even after cabal has a -fforce-recomp build in the cache?

Reply:
The purpose of the flag is not to cause cabal to always rebuild the file. The purpose is to rebuild the file if cabal invokes ghc on it.